source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -68,24 +68,33 @@ To remove a library dependency from a project, select :guilabel:`Manage Current
 Libraries
 ---------
 
-.. warning:: These are **not** links to directly plug in to the :guilabel:`VS Code` -> :guilabel:`Install New Libraries (online)` feature. Click these links to visit the vendor site to see whether they offer online installers, offline installers, or both.
+.. warning:: Many vendors (like CTRE & REV) with additional programs to aid in the use of thier libraries prefer the offline installer.  Click these links to visit the vendor site to see whether they offer online installers, offline installers, or both.
```

Comment:
I think the first sentence is the warning. The second sentence is more instructions and should probably be removed from the warning so it doesn't get lost.